Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx

```diff
@@ -514,7 +514,7 @@ void vtkMRMLApplicationLogic::PropagatePlotChartSelection()
 }
 
 //----------------------------------------------------------------------------
-void vtkMRMLApplicationLogic::FitSliceToAll(bool onlyIfPropagateVolumeSelectionAllowed /* =false */)
+void vtkMRMLApplicationLogic::FitSliceToAll(bool onlyIfPropagateVolumeSelectionAllowed /* =false */, bool rotationAllowed /* =true */)
```

Comment:
`rotationAllowed` is not a good name here. It is not about it being allowed (as opposed to the other argument, which handles the case when slice composite nodes do not allow propagate volume selection), but being applied. I suggest `resetOrientation` instead.

Reply:
Done